.github/workflows/plugins-test.yaml

```diff
@@ -492,3 +492,28 @@ jobs:
         run: ./mvnw -f test/plugin/pom.xml clean package -DskipTests docker:build -DBUILD_NO=local >/dev/null
       - name: Run mysql 5.1.2-8.0.15 (53)
         run: bash test/plugin/run.sh mysql-scenario
+
+  Pulsar:
```

Comment:
Pulsar test is quick enough, please add it into `Feign_Canal` group, and rename it to `Feign_Canal_ Pulsar`.

Reply:
Sure

Reply:
sure